command/agent/agent_endpoint_test.go

```diff
@@ -1634,17 +1625,17 @@ func TestHTTP_AgentSchedulerWorkerRequest_NoACL(t *testing.T) {
 	configFn := func(c *Config) {
 		var numSchedulers = 8
 		c.Server.NumSchedulers = &numSchedulers
-		c.Server.EnabledSchedulers = []string{"_core", "system"}
+		c.Server.EnabledSchedulers = []string{"_core", "batch"}
 		c.Client.Enabled = false
 	}
 	testFn := func(s *TestAgent) {
 		for _, tc := range schedulerWorkerTest_testCases() {
 			t.Run(tc.name, func(t *testing.T) {
 
-				req, err := http.NewRequest(tc.request.verb, "/v1/agent/workers", nil)
+				req, err := http.NewRequest(tc.request.verb, "/v1/agent/workers", bytes.NewReader([]byte(tc.request.requestBody)))
```

Comment:
Could you add this configuration to the OpenAPI project? Or at the very least raise an issue there so it can be updated once this gets released?

Reply:
@DerekStrickland thanks for making the tracking issue for this.